x-pack/legacy/plugins/apm/server/lib/traces/get_trace_items.ts

```diff
@@ -34,7 +35,9 @@ export async function getTraceItems(
         bool: {
           filter: [
             { term: { [TRACE_ID]: traceId } },
-            { terms: { [PROCESSOR_EVENT]: ['span', 'transaction'] } },
+            {
+              terms: { [PROCESSOR_EVENT]: ['span', ProcessorEvent.transaction] }
```

Comment:
Please add span as a processor event as well:

```suggestion
              terms: { [PROCESSOR_EVENT]: [ProcessorEvent.span, ProcessorEvent.transaction] }
```

Reply:
I am running into a little issue. 

In order to make this change, I first added a 'span' member to the ProcessorEvent enum (in file `processor_event.ts`). I then went ahead and added span as a processor event as instructed here. However, after doing so, I am getting an error in file `get_dynamic_index_pattern.ts` displayed in the image below. Is there something I am missing? 

![image](https://user-images.githubusercontent.com/28423205/71144872-845eea00-21ed-11ea-8a0a-d4953dd1623e.png)

Reply:
I think it's fine to add `span` to `indicesMap` simply to satisfy typescript:

```ts
  const indicesMap = {
    transaction: indices['apm_oss.transactionIndices'],
    metric: indices['apm_oss.metricsIndices'],
    error: indices['apm_oss.errorIndices'],
    span: indices['apm_oss.spanIndices']
  };
```

Reply:
Done.